index.js

```diff
@@ -282,6 +284,7 @@ Client.prototype.add = function (info) {
         });
     } else if ((typeof info === 'string') || (info instanceof Object && !(info instanceof StreamReadable) && !(info instanceof StreamWritable))) {
         // We've been passed single server information
+        var stream = [];
```

Comment:
this doesn't make any difference as `var` declarations get lifted to the top of the function either way